### ISSUE TITLE
Search can return more than 50 metabolites

### ISSUE THREAD

**Issue by e0** [MetabolicAtlas/MetabolicAtlas#848]
**Describe the bug**

The quick search endpoint should return max 50 results per type. Currently it is possible to return more than 50 metabolites. There is a failing test for this.

**Steps to reproduce**

Run the tests for `api`: `ma-exec api yarn test`. Verify that there is a failing test where more than 50 metabolites are returned.

**Expected behavior**

Make sure that the relevant failing test can pass.

**Additional context**

The test is failing now because the results from `metabolite` and `compartmentalizedMetaboltie` are combined. There is a max limit of 50 for each of them so by combining them it could exceed 50.

**Comment by perjo** [comment]
> Run the tests for api: ma-exec api yarn test. Verify that there is a failing test where more than 50 metabolites are returned.

Which test is this? I backed to before #1115 was merged but I could not find a failing test matching this description? @e0

**Comment by e0** [comment]
@perjo I don't remember when this was fixed. I think it's [this test](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/main/api/test/searchGem.test.js#L9).

**Comment by perjo** [comment]
Ok, a bit of confusion here. Seems problem is fixed by [limiting the number of results per type in the frontend](https://github.com/MetabolicAtlas/MetabolicAtlas/commit/e7a3bb575f2a4a0bb6d61756c9135b497cef915f). (this number has changed back and forth since and is now 10). 

As to why test was failing before it seems the test may have been incorrectly written at first 

`expect(metabolite.length).toBeLessThan(50);` (3b3b01e5388e2fce301f841f824e350a2ddc32f9)

and was got to pass by 

`expect(metabolite.length).toBeLessThanOrEqual(50)` (c3594ea23eff2db5e7d0b1ea2f483f5798823e5b)

To sum up, it seems this issue is fixed in the frontend and not in the API as first suggested, but I guess that is fine and we can close it?

**Comment by e0** [comment]
We discussed this a bit more in #968. I think it is still possible for the results to be greater than the limit, but that is probably not something we want to fix since it has a positive side effect for the frontend. I think we could either leave things as is or even remove this test since it's not something we currently plan to fix.
